src/main.py

```diff
@@ -300,6 +299,7 @@ def run(self):
                                     sub.sticky()
                                     print "Submission stickied..."
                             break
+                        else: time.sleep(120)

```

Comment:
This is outside the game thread while loop on like 250, so this is going to constantly do this and definitely make us hit the reddit API rate limit quickly.

The rate limit has gone up recently though (since oauth was implemented, it's 60 req/min), so I'd say we should just change the sleep to like 10-30 seconds. We could also build this time dynamically using the `X-Ratelimit` vars sent back by the reddit api, but I don't know if we can do that through praw.